internal/cli/atlas/clusters/create.go

```diff
@@ -232,7 +232,12 @@ Some of the cluster configuration options are available via flags but for full c
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return opts.Run()
 		},
+		Annotations: map[string]string{
+			"args":     "name",
```

Comment:
how to get the args and descriptions from a command, we'll need to back fill this